src/esrs/aggregate.rs

```diff
@@ -25,7 +27,7 @@ pub trait Aggregate {
 
     /// An event represents a fact that took place in the domain. They are the source of truth;
     /// your current state is derived from the events.
-    type Event: Send + Sync;
+    type Event: Send + Sync + Debug;
```

Comment:
I'm not sure i want to add this bound forcing user to implement debug in its events only to debug `StoreEvent`. I would say that the "important" part of the store event is the payload and would be possible anyway to debug the payload if in your codebase you derive debug on that. Moreover all the other fields are debuggable. Maybe i'm missing the main reason of this change?

Reply:
Yep I thought the same thing about the enforcing of `Debug` but then everything that is saved inside an event should be "debuggable" by itself. I mean, I cannot think of anything that could no be debuggable.

The main PRO that you get with it is that it's much easier to "just" introspect the whole stored events with their sequence number & aggregate ids. Without it you're forced to extract these fields by hand.

P.S: I faced the need to check the whole stored events while debugging some aggregate unit tests that were breaking due to unmet db uniqueness constraints 😓